### PR TITLE
Fixes #18923: Missing API addition for version 13 in doc

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/18923

### CODE REVIEW

File: webapp/sources/api-doc/introduction.md [hunk 1]
```diff
@@ -62,9 +62,16 @@ period of time to allow migration from previous versions.
...
     <tr>
       <td class="code">12</td>
-      <td class="code">6.0</td>
+      <td class="code">6.0 and 6.1</td>
       <td>Node key management</td>
     </tr>
+    <tr>
+      <td class="code">13</td>
+      <td class="code">6.2</td>
+      <td>Node status endpoint</td>
```
Comment (amousset) [inline webapp/sources/api-doc/introduction.md:71]: there are only three column in this table, you can use an ul inside of the td

**Comment by fanf** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Comment by amousset** [comment]
OK, merging this PR

**Comment by amousset** [comment]
OK, merging this PR

**Pull request merged**